### DISCUSSION TITLE
Possibility to pin documents

### DISCUSSION THREAD

**Discussion by jogboms** [notaapp/talk#215] [Feature Requests]
It would be nice i think to be able to pin certain documents within directories to the top for easy access.

**Comment by astoilkov** [comment]
An interesting idea, we will explore it. Thanks.

BTW, have you considered adding a number or a special symbol like `_` before the folder name to put it on top? This way it will be on top in Finder and forever, regardless of the software you use. You can also experiment with [Johnny Decimal numbering system](A system to organize projects).

**Comment by fedorovpishet** [comment]
> BTW, have you considered adding a number or a special symbol like `_` before the folder name to put it on top? This way it will be on top in Finder and forever, regardless of the software you use. You can also experiment with [Johnny Decimal numbering system](A system to organize projects).

I use numbers or a special symbols to organise notes. But sometimes I need pin some files from different folders in one place. Maybe texts for an actual project. I don't need hold it always on top but only temporarily. It's very useful feature.
<img width="1311" alt="image" src="https://user-images.githubusercontent.com/90445152/165734729-bf415beb-4ad6-4d3e-b340-519702979d8d.png">

**Comment by astoilkov** [comment]
@fedorovpishet That's interesting. So we have two possible implementations emerging. One is to pin documents to the top of folders and the other is to pin them globally regardless of the folder. What do you think about pinned tabs?

**Comment by fedorovpishet** [comment]
I like use tabs in Nota, but it does't sounds comfortable to pin them. We do not pin tabs in Safari, right? We use Favorites. And there is the same in other note-taking apps. 

Here's how it looks in WorkFlowy for instance.

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/90445152/165922160-c3be8aef-397f-43f3-8eb8-cf0a41609b57.png">

Or Notion.

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/90445152/165922412-3ebac325-3583-473b-b37b-0115e1269f51.png">

 Roam, Obsidian, Ulysses etc. They all use side panel for pined and favorites notes. 

So I guess "to pin documents to the top of folders" will be the best solution.

**Reply by bellebethcooper** [reply]
I do pin tabs in Safari, and in my IDE. It would be handy to be able to pin tabs so they stay visible when I switch workspaces, since I have reference notes and checklists in my Nota folder, and sometimes want to reference them while working on a file in another project, so I have to keep switching workspaces.

**Reply by astoilkov** [reply]
We are currently working on pins. We may even release them in the next release.

**Comment by webwesen** [comment]
my vote goes to global pins in the left nav (i.e. Logseq)

**Comment by astoilkov** [comment]
We started experimenting with a little redesign that will allow us to pin and have favorites. I will post updates here. Thanks for all the feedback!

**Comment by fedorovpishet** [comment]
BTW, iA Writer got a very cool similar feature — favorites notes and folders with hot-keys.

How it works. You set a note or a folder as favorite in context menu.

<img width="733" alt="image" src="https://user-images.githubusercontent.com/90445152/167082313-3e23d782-c372-47a0-9786-405a5438d562.png">

And now you can open your favorite note or folder by control+1....9. It's SO useful and I don't understand why no one another note-taking app use this feature too. 🤯 

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/90445152/167081520-52beee0f-a364-4bfb-855d-89aa51b88706.png">

Sorry about screenshots in Russian. I can't find how to change interface language 😊

**Comment by astoilkov** [comment]
Wow, that's great. I was thinking about the same thing. If there are favorites, they should be available through a shortcut as well. I was thinking about custom shortcuts but I like this better.
